bin/azurite/wait

```diff
@@ -23,7 +23,7 @@ healthy() {
 }
 
 until healthy "$(curl --silent --max-time 2 "${azurite_endpoint}/")"; do
-  if (( "$(date +%s)" > "${DEADLINE}" )); then
+  if (("$( date +%s)" > "${DEADLINE}")); then
```

Comment:
question (non-blocking): the space before `date` here looks a little funny to me, did the formatter do this to get it to match the style guide?

Reply:
Hmmm

Reply:
OK, it's done now.

Reply:
(It was a bug in the -kp flag : https://github.com/mvdan/sh/issues/362)